sdk/servicebus/test-resources.json

```diff
@@ -0,0 +1,220 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "baseName": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().name]",
+      "metadata": {
+        "description": "The base resource name."
+      }
+    },
+    "subscriptionId": {
+      "type": "string",
+      "defaultValue": "[subscription().subscriptionId]",
+      "metadata": {
+        "description": "The subscription ID to which the application and resources belong."
+      }
+    },
+    "tenantId": {
+      "type": "string",
+      "defaultValue": "[subscription().tenantId]",
+      "metadata": {
+        "description": "The tenant ID to which the application and resources belong."
+      }
+    },
+    "testApplicationOid": {
+      "type": "string",
+      "metadata": {
+        "description": "The client OID to grant access to test resources."
+      }
+    },
+    "testApplicationId": {
+      "type": "string",
+      "metadata": {
+        "description": "The application client ID used to run tests."
+      }
+    },
+    "testApplicationSecret": {
+      "type": "string",
+      "metadata": {
+        "description": "The application client secret used to run tests."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "The location of the resources. By default, this is the same as the resource group."
+      }
+    }
+  },
+  "variables": {
+    "serviceBusNamespace": "[concat('sb-', parameters('baseName'))]",
+    "serviceBusSubscriptionName": "[concat('sb-', parameters('baseName'), '-subscription')]",
+    "serviceBusTopicName": "[concat('sb-', parameters('baseName'), '-topic')]",
+    "serviceBusQueueName": "[concat('sb-', parameters('baseName'), '-queue')]",
+    "serviceBusSessionQueueName": "[concat('sb-', parameters('baseName'), '-session-queue')]",
+    "sessionQueueAuthorizationRuleName": "[concat('sb-', parameters('baseName'), '-session-queue-auth')]",
+    "serviceBusSessionId": "[concat('sb-', parameters('baseName'), '-session-queue-id')]",
+    "defaultSASKeyName": "RootManageSharedAccessKey",
+    "authRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', variables('serviceBusNamespace'), variables('defaultSASKeyName'))]",
+    "sbVersion": "2017-04-01",
+    "contributorRoleId": "b24988ac-6180-42a0-ab88-20f7382dd24c",
+    "serviceBusDataOwnerRoleId": "090c5cfd-751d-490a-894a-3ce6f1109419"
```

Comment:
same question as the event hub PR -- are these two ids public constants?

Reply:
yes here: https://docs.microsoft.com/en-us/azure/role-based-access-control/built-in-roles

Reply:
thanks!